### ISSUE TITLE
Storage query not returning expected results

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-model#130]
@jfpucheu is seeing poorly formed results from one of the cluster costs queries. First step is that we need better logging to determine exactly what is going on.

**Comment by dwbrown2** [comment]
Here's the query:

```
(sum(kube_persistentvolumeclaim_info) by (persistentvolumeclaim, storageclass) + on (persistentvolumeclaim)
           group_right(storageclass) sum(kubelet_volume_stats_used_bytes) by (persistentvolumeclaim))
```

Error executing query: `many-to-many matching not allowed: matching labels must be unique on one side`

**Comment by dwbrown2** [comment]
This can be caused by PV with the same name in different namespaces... updated query:

```(sum(kube_persistentvolumeclaim_info) by (persistentvolumeclaim, storageclass,namespace) + on (persistentvolumeclaim,namespace)
           group_right(storageclass) sum(kubelet_volume_stats_used_bytes) by (persistentvolumeclaim,namespace))```

**Comment by dwbrown2** [comment]
Fixed, will be in the v1.35 build.
